src/components/Envelope.vue

```diff
@@ -903,7 +903,6 @@ export default {
 			text-overflow: ellipsis;
 			white-space: nowrap;
 			line-height: 130%;
-			display: flex;
 			align-items: center;
```

Comment:
```suggestion
```

https://developer.mozilla.org/en-US/docs/Web/CSS/align-items this is a flexbox prop. Make sure elements are still aligned the same way.

Reply:
yes, the elements are aligned in the center in all layouts

Reply:
good. then let's just drop the dead prop